dojo/1.11/dojo.d.ts

```diff
@@ -784,8 +784,8 @@ declare namespace dojo {
 	/* dojo/Evented */
 
 	interface Evented {
-		on(type: string | ExtensionEvent, listener: EventListener): Handle;
-		emit(type: string | ExtensionEvent, event: any): boolean;
+		on(type: string | ExtensionEvent, listener: EventListener | Function): Handle;
+		emit(type: string | ExtensionEvent, ...events: any[]): boolean;
```

Comment:
Using an ES6 rest parameter means that you can supply an arbitrary number of event objects, or none at all. The following would be valid use cases:

```javascript
emit("event");
emit("event", obj);
emit("event", obj1, obj2, obj3);
```